services/storage/src/simcore_service_storage/datcore_adapter/datcore_adapter.py

```diff
@@ -0,0 +1,192 @@
+import asyncio
```

Comment:
STYLE: why folder and prefix  i.e. ``datcore_adapter/datcore_adapter_*`` ?  I would suggest one or the other but not both.

Reply:
yes you are right. but when I look for files in vscode, I like it when I find them right away. If I look for ```client.py``` then I will get 10 entries. if I look for datcore_adapter.py I get 1.
so I don't know. let's talk tomorrow.

Reply:
vscode adds now also the folder it belongs to ...  :-)